.github/CODEOWNERS

```diff
@@ -166,7 +166,8 @@ packages/react-components/react-shared-contexts/library @microsoft/teams-prg @mi
 packages/react-components/react-shared-contexts/stories @microsoft/teams-prg @microsoft/cxe-red
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31600&commitId=cbe7a863cef1ae7e92f111c1c225365627780583&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>